src/app/recommendations.ts

```diff
@@ -59,6 +59,7 @@ export const RECOMMENDATIONS: Step[] = [
   { possibleIn: 700, necessaryAsOf: 700, level: 2, material: true, step: 'v7 material changes', action: 'If you use screenshot tests, you\'ll need to regenerate your screenshot golden files as many minor visual tweaks have landed.'},
   { possibleIn: 700, necessaryAsOf: 800, level: 3, material: true, step: 'v7 material deprecations', action: 'Stop using `matRippleSpeedFactor` and `baseSpeedFactor` for ripples, using Animation config instead.'},
   { possibleIn: 800, necessaryAsOf: 800, level: 1, step: 'v8 update', action: 'Update to version 8 of the core framework and CLI by running `ng update @angular/cli @angular/core` in your terminal'},
+  { possibleIn: 800, necessaryAsOf: 800, level: 1, step: 'use ::ng-deep instead of /deep/', action: 'Replace `/deep/` with `::ng-deep` in your styles, [read more about angular component styles and ::ng-deep](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html). `/deep/` and `::ng-deep` both are deprecated but using `::ng-deep` is preferred until the shadow-piercing descendant combinator is [removed from browsers and tools](https://www.chromestatus.com/features/6750456638341120) completely.', },
```

Comment:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html doesn't seem to have anything to do with "angular component styles and ::ng-deep". Ideally, Angular should be capitalized.

I think that the link should be https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep?